### PR TITLE
refactor: extracted worker queue chunk size to env var

### PR DESCRIPTION
**Pull request by Khongchai**
### What change does this PR introduce?

Extract hard-coded queue size into env var.

### Why was this change needed?

To make it easier for people who self-host to optimize queue size.

### Other information

https://discord.com/channels/895029566685462578/1185606056005611642/1185861985808695336

### CODE REVIEW

**Comment by Khongchai** [comment]
@scopsy

File: packages/application-generic/src/usecases/trigger-broadcast/trigger-broadcast.usecase.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ import { SubscriberProcessQueueService } from '../../services/queues/subscriber-
 import { TriggerBroadcastCommand } from './trigger-broadcast.command';
 
 const LOG_CONTEXT = 'TriggerBroadcastUseCase';
-const QUEUE_CHUNK_SIZE = 100;
+const QUEUE_CHUNK_SIZE = process.env.QUEUE_CHUNK_SIZE ?? 100;
```
Comment (scopsy) [inline packages/application-generic/src/usecases/trigger-broadcast/trigger-broadcast.usecase.ts:33]: I wonder should we maybe have 2 separate more specific names here?

```suggestion
const QUEUE_CHUNK_SIZE = process.env.BROADCAST_QUEUE_CHUNK_SIZE ?? 100;
```

and for the multicast `MULTICAST_QUEUE_CHUNK_SIZE` wdyt?

Comment (Khongchai) [reply packages/application-generic/src/usecases/trigger-broadcast/trigger-broadcast.usecase.ts:33]: I agree! Will apply your suggestion

Comment (Khongchai) [reply packages/application-generic/src/usecases/trigger-broadcast/trigger-broadcast.usecase.ts:33]: I also forgot to parse int from process env string. Will add parseInt as well

**Comment by punchvv** [comment]
Any update on this PR?

**Review by scopsy — APPROVED** [review]

**Pull request merged**